src/features/loading/LoadingScreen.tsx

```diff
@@ -120,8 +121,8 @@ class LoadingScreen extends Component<TLoadingProps, TLoadingState> {
 
   loadingConfigs = async () => {
     const success = await this.ensurePastelParams()
-
     if (success) {
+      initServeStatic(remote.app.isPackaged)
```

Comment:
I think you can call ipcMain here and dispatch "loadConfigSuccess" event, and in the `main.ts`, listen for that event and call `initServeStatic(remote.app.isPackaged)`. It would be cleaner. Read more about ipcMain here: https://www.electronjs.org/docs/api/ipc-main

Reply:
It has been updated